interfaces/cython/cantera/lxcat2yaml.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# This file is part of Cantera. See License.txt in the top-level directory or
+# at https://cantera.org/license.txt for license and copyright information.
+
+"""
+lxcat2yaml.py: Convert the LXCat integral cross-section data in XML format to YAML format.
+The cross-section data is used to calculate the reaction rate of a electron-collision
+process in a plasma. The data can be downloaded at https://nl.lxcat.net/data/xml/lxcat_xml.zip.
+
+Usage:
+    lxcat2yaml [--input=<filename>]
+               [--database=<database name>]
+               [--mech=<filename>]
+               [--phase=<phase name>]
+               [--insert]
+               [--output=<filename>]
+
+Example:
+    lxcat2yaml --input=mycs.xml --database=itikawa --mech=oxygen-plasma.yaml
+               --phase=isotropic-electron-energy-plasma --insert
+               --output=oxygen-itikawa-plasma.yaml
+
+Option --database=<database name> is required. The database needs to be selected
+and its name can be found in the XML file.
```

Comment:
The XML file has multiple databases. For example Lisbon and Itikawa both have oxygen collision data. User needs to determine which data to be used.

Reply:
Sure, but each of these is stored in a separate file, which the user is already specifying as an argument, right? As far as I can tell, there is only one `<database>` tag in each LXCAT file, although perhaps the format technically supports multiple entries. In the case where there is only one `<database>` tag, it would be more user-friendly to use it automatically.